docs/examples/frameworks/jax/job.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#SBATCH --gpus-per-task=rtx8000:1
+#SBATCH --cpus-per-task=4
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=16G
+#SBATCH --time=00:15:00
+
+
+# Echo time and hostname into log
+echo "Date:     $(date)"
+echo "Hostname: $(hostname)"
+
+
+# Ensure only anaconda/3 module loaded.
+module --quiet purge
+# This example uses Conda to manage package dependencies.
+# See https://docs.mila.quebec/Userguide.html#conda for more information.
+module load anaconda/3
+
+# Creating the environment for the first time:
+# conda create -y -n jax -c "nvidia/label/cuda-11.8.0" cuda python=3.9 virtualenv pip
+# conda activate jax
+# Install Jax using `pip`
+# *Please note* that as soon as you install packages from `pip install`, you
+# should not install any more packages using `conda install`
+# pip install --upgrade "jax[cuda11_pip]" \
+#    -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+# Other pip packages:
+# pip install pillow optax rich torch torchvision flax tqdm
```

Comment:
This one? https://mila-umontreal.slack.com/archives/CRLBKSFJ9/p1684349474001709?thread_ts=1684332902.685989&cid=CRLBKSFJ9

Reply:
No @obilaniu didn't had the time to forward me the info yet but here I explain how I ended up with this method so it could be modified with another CUDA version : [docs/examples/frameworks/jax_setup/_index.rst](https://github.com/mila-iqia/mila-docs/pull/205/files/45e34d4c5f14e8a45c17c45fa540092cd5497112#diff-27c6aaddc3c615c9273fc87074d91d63555259d6ff5a6f441f327e32308cba62R29-R75)

I've only tried with the recommended way [to install with Conda](https://github.com/google/jax#conda-installation) but I don't understand the tuning @obilaniu has in :

```
CONDA_OVERRIDE_CUDA=11.7 conda create -n py39jax --override-channels -c nvidia -c conda-forge python=3.9 cudatoolkit=11.7 cuda-nvcc cudnn jax 'jaxlib=*=cuda112py*'
```

When he will be able to have a look I think we'll be able to decide which one is simpler for the user